data-sdk/src/main/java/com/logicmonitor/sdk/data/Configuration.java

```diff
@@ -59,12 +61,15 @@ public Configuration() {
    * @param accessId
    * @param accessKey
    * @param bearerToken
+   * @param domainName
    */
-  public Configuration(String company, String accessId, String accessKey, String bearerToken) {
+  public Configuration(
+      String company, String accessId, String accessKey, String bearerToken, String domainName) {
     this.company = (company != null) ? company : System.getenv("LM_COMPANY");
     this.accessId = (accessId != null) ? accessId : System.getenv("LM_ACCESS_ID");
     this.accessKey = (accessKey != null) ? accessKey : System.getenv("LM_ACCESS_KEY");
     this.bearerToken = (bearerToken != null) ? bearerToken : System.getenv("LM_BEARER_TOKEN");
+    this.domainName = (domainName != null) ? domainName : System.getenv("LM_DOMAIN_NAME");
```

Comment:
For the other SDKs, are we following a similar approach, such as adding new environment variables?